code/modules/backgrounds/location/_location_events.dm

```diff
@@ -34,12 +34,12 @@
 
 /decl/location_event/alien_raiders/announce(var/decl/background_detail/location)
 	if(prob(20))
-		. = "The Tiger Co-operative have raided [location.name] today, no doubt on orders from their enigmatic masters. Stealing wildlife, farm animals, medical research materials and kidnapping civilians. [global.using_map.company_name] authorities are standing by to counter attempts at bio-terrorism."
+		. = "The Tiger Co-operative have raided [location.name] today, no doubt on orders from their enigmatic masters, stealing wildlife, farm animals, medical research materials and kidnapping civilians. [global.using_map.company_name] authorities are standing by to counter attempts at bio-terrorism."
```

Comment:
```suggestion
		. = "The Tiger Cooperative have raided [location.name] today, no doubt on orders from their enigmatic masters, stealing wildlife, farm animals, medical research materials and kidnapping civilians. [global.using_map.company_name] authorities are standing by to counter attempts at bio-terrorism."
```
co-operative is BrEng, cooperative is AmEng, not sure if we have a preference here